etc/ProdOfflineConfiguration.py

```diff
@@ -582,10 +582,17 @@
                dqm_sequences=["@common", "@muon", "@heavyFlavor"],
                alca_producers=["TkAlJpsiMuMu", "TkAlUpsilonMuMu"],
                physics_skims=["LogError", "LogErrorMonitor", "BPHSkim", "MuonPOGJPsiSkim"],
-               tape_node="T1_US_FNAL_MSS",
+               tape_node=None,
                disk_node="T1_US_FNAL_Disk",
                scenario=ppScenario)
 
+DATASETS = ["ParkingDoubleMuonLowMass"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               scenario=ppScenario)
+
```

Comment:
If this is the one that will be prompt-recoed (instead of `DoubleMuonLowMass`) then we need to add the `alca_producers` and `dqm_sequences` to rhis PD.

Reply:
Done with https://github.com/dmwm/T0/pull/4716/commits/8e3569fff183b26dbb6e4ff62233ececf8fd2777